src/main/java/elberger/paint/PaintFrame.java

```diff
@@ -0,0 +1,65 @@
+package elberger.paint;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionListener;
+
+public class PaintFrame extends JFrame
+{
+	private Canvas canvas;
+	private JButton chooseColor;
+	private JButton choosePencil;
+	private JButton chooseRectangle;
+	private Color color;
+
+	public PaintFrame()
+	{
+		setTitle("Paint");
+		setSize(800, 600);
+		setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+		JPanel panel = new JPanel();
+		panel.setLayout(new BorderLayout());
+
+		JPanel buttonPanel = new JPanel();
+
+		chooseColor = new JButton("Choose Color");
+		choosePencil = new JButton("Pencil");
+		chooseRectangle = new JButton("Rectangle");
+
+		buttonPanel.add(chooseColor);
+		buttonPanel.add(choosePencil);
+		buttonPanel.add(chooseRectangle);
+
+		ActionListener choosingColor = e ->
```

Comment:
`e` is usually reserved for Exceptions
You can put all this in the `addActionListener` method, instead of creating an ActionListener variable.